tests/test_marleyspoon.py

```diff
@@ -0,0 +1,104 @@
+        self.assertEqual(
+            {
+                "calories": "1143kcal",
+                "carbs": "103.8g",
+                "fat": "68.0g",
+                "proteins": "27.8g",
+            },
+            self.harvester_class.nutrients(),
+        )
+
+    def test_ingredients(self):
+        self.assertEqual(
+            [
+                "25g Erdnüsse, geröstet & gesalzen",
+                "2 vegane Burgerpattys",
+                "2 Burgerbrötchen mit Sesam",
+                "1 Päckchen Sriracha-Sauce",
+                "25ml Sojasauce",
+                "2 Stangensellerie",
+                "1 Minigurke",
+                "1 Apfel",
+                "1 Packung festkochende Kartoffeln",
+                "2EL Mayonnaise",
+                "2½TL Honig",
+                "½TL Senf",
+                "Salz",
+                "Zucker",
+                "Olivenöl",
+                "Essig",
+            ],
+            self.harvester_class.ingredients(),
+        )
+
+    def test_instructions(self):
+        return self.assertEqual(
+            "Den Backofen auf 240°C (220°C Umluft) vorheizen. Die __Kartoffeln samt Schale__ in ca. 1cm breite Spalten schneiden und auf einem mit Backpapier ausgelegten Backblech mit 1EL Olivenöl sowie 1/2TL Salz vermengen. Die __Kartoffeln__ im Ofen auf mittlerer Schiene 20–25Min. goldbraun rösten, zwischendurch einmal wenden.\n2EL Olivenöl, 1EL hellen Essig, 1/2TL Senf, 1/2TL Honig und 1 kräftige Prise Salz zu einem __Dressing__ verrühren. Den __Apfel__ vierteln, entkernen und in ca. 1cm große Würfel schneiden. Die __1/2 der Gurke__ ebenfalls in ca. 1cm große Würfel schneiden. Den __Sellerie__ in dünne Scheiben schneiden. Die __Apfel-__ und __Gurkenwürfel__ und den __Sellerie__ mit dem __Dressing__ vermengen.\nDie __restliche Gurke__ mit einem Sparschäler in breite Streifen schneiden. 1EL hellen Essig, 1TL Zucker und 1 kräftigen Prise Salz verrühren, bis sich Zucker und Salz aufgelöst haben. Mit den __Gurkenstreifen__ vermengen und beiseitestellen.\nDie __Sojasauce__ mit 2TL Honig und 1EL hellem Essig verrühren. Getrennt davon die __Sriracha-Sauce__ mit 2EL Mayonnaise verrühren. __Tipp:__ Wenn Kinder mitessen, ggf. weniger __Sriracha-Sauce__ verwenden oder diese separat servieren. Die __Burgerbrötchen__ auf einem Backrost 2–3Min. im Ofen aufbacken.\nDie __Pattys__ in einer mittelgroßen Pfanne mit 1EL Olivenöl bei mittlerer Hitze auf jeder Seite 1–2Min. goldbraun braten. Die __Soja-Würzsauce__ dazugeben, 1–2Min. köcheln lassen und die __Pattys__ in der __Glasur__ wenden, sodass sie vollständig damit bedeckt sind.\nDie __Brötchen__ aufschneiden, mit dem __Dip__ bestreichen und nach Geschmack mit den __Pattys__, den __Gurkenstreifen__ und je ca. __1EL Apfel-Gurken-Salat__ belegen. Den __restlichen Salat__ mit den __Erdnüssen__ bestreuen und zu den __Burgern__, den __Röstkartoffeln__ und ggf. __übrigen Gurkenstreifen__ servieren. Ggf. __übrigen Dip__ oder mehr Mayonnaise nach Geschmack dazu reichen.",
```

Comment:
Hmm.. one more thought before approving this: I think we should probably try to trim the surrounding double-underscores from the instructions.

They're kinda useful to identify the ingredient names, but they'd also be a bit distracting and confusing for someone who had loaded recipes into a recipe app using this library.

Reply:
Done